### DISCUSSION TITLE
Missing title or filename

### DISCUSSION THREAD

**Discussion by MarlonVisser** [TagStudioDev/TagStudio#619] [Q&A] [answered]
I've looked around a lot to see if there's anything I can change to let the Title or Filename show up. I want to use TagStudio to organize and tag through complicated relational data. It seems that it doesn't show filenames, which makes it impossible for me to use. I'm only 1 feature away from being able to use this to my full wishes.

Sorry if this is double, I've tried my honest best to see if there's other questions like this.

![image](https://github.com/user-attachments/assets/d9a91daa-c18c-4848-b1a1-3b72415aeb85)

Thank you <3

**Comment by CyanVoxel** [comment, marked as answer]
Hello! Unfortunately there's no way to do this in the program currently, but it's something we would like to add: https://github.com/TagStudioDev/TagStudio/issues/85

**Reply by MarlonVisser** [reply]
Thank you. I totally missed it with my searches. Thank you for the quick response, you're great! I'll wait patiently so I can start using this. It's suits my needs perfectly, I'll roll the dice and just wait out the functionality.
